tests/test_elasticsearch.py

```diff
@@ -79,22 +79,46 @@ def test_elasticsearch_with_sysctl_disabled(self, kube_version):
         )
 
         assert len(docs) == 3
-        doc = docs[0]
 
         # elasticsearch master
+        doc = docs[0]
         assert doc["kind"] == "StatefulSet"
-        assert "sysctl" not in jmespath.search(
-            "spec.template.spec.initContainers[*].name", docs[0]
-        )
+        assert jmespath.search("spec.template.spec.initContainers", docs[0]) is None
```

Comment:
Shorter and maybe more legible:
```suggestion
        assert not docs[0]['spec']['template']['spec']['initContainers']
```

Or you could wrap all of these up in:

```
assert not any([doc['spec']['template']['spec']['initContainers'] for doc in docs])
```